### PR TITLE
Add support for importing from Authy

### LINKED ISSUE #65: Support importing from Authy

### PR DESCRIPTION
**Pull request by michaelschattgen**
This pull request adds the ability to import entries from Authy, which requires root. I had to do some ugly string hacks in order to set the correct issuer/account name. Authy's database files are a mess.

Closes #65

### CODE REVIEW

File: app/src/main/java/com/beemdevelopment/aegis/importers/DatabaseImporter.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
         _appImporters.put("FreeOTP", FreeOtpImporter.class);
         _appImporters.put("Google Authenticator", GoogleAuthImporter.class);
         _appImporters.put("Steam", SteamImporter.class);
+        _appImporters.put("Authy", AuthyImporter.class);
```
Comment (alexbakker) [inline app/src/main/java/com/beemdevelopment/aegis/importers/DatabaseImporter.java]: Let's keep these lists sorted alphabetically. Can you also add it to the list of regular importers?

**Comment by julianxhokaxhiu** [comment]
May I ask, as there is no documentation about this feature, how am I supposed to use the import from file function as import from app is only possible when you have a rooted phone?

**Comment by michaelschattgen** [comment]
Hi Julian, it is possible to extract Authy's database file from a rooted phone. This feature just allows you to import this file whether your phone is rooted or not.

If you don't want to root the device you installed Aegis on, you can always:

1. Root another device (even emulator maybe?)
2. Install Authy
3. Export Authy database file
4. Import that file on your non-rooted device with Aegis

**Comment by julianxhokaxhiu** [comment]
Thanks, now it makes more sense to me :) I'll find my way out, I thought i was missing some steps.

**Comment by michaelschattgen** [comment]
No worries! Feel free to contact us when you have issues or questions.

**Comment by digitalTrilunaire** [comment]
Hi Guys, just discover that this method work (so no root and no emulator to install) https://gist.github.com/gboudreau/94bb0c11a6209c82418d01a59d958c93#gistcomment-3240205 . Codes are identicals :)

**Comment by firozzer** [comment]
Authy database file location: `/data/data/com.authy.authy/shared_prefs/com.authy.storage.tokens.authenticator.xml,`

For future searchers.

**Pull request merged**